README.rst

```diff
@@ -1,11 +1,13 @@
-================================
+﻿================================
   MDAnalysis Repository README
 ================================
 
 |numfocus| |build| |cron| |travis| |cov| [*]_
 
 |docs| |devdocs| |usergroup| |developergroup| |anaconda| |mybinder|
 
+This is my repository
```

Comment:
@yogesh7025 can you explain what you are trying to do here? This doesn't seem to attempt to fix anything.

Reply:
Ohh, I am sorry, I am new to GitHub, i forked mdanalysis to my GitHub account and was playing with it by cloning it in my GitHub desktop, I added some text to check how to merg, I thought it will only merge with the copy of my mdanalysis (the copy on my  GitHub account) but now the request has gone to you people!!

Sorry again. 

I will try to figure out how to do stuff locally.
Peace.